pkg/cmd/attestation/verification/extensions.go

```diff
@@ -46,5 +51,26 @@ func verifyCertExtensions(attestation *AttestationProcessingResult, tenant, owne
 		}
 	}
 
+	// if issuer is anything other than the default, use the user-provided value;
+	// otherwise, select the appropriate default based on the tenant
+	if issuer != GitHubOIDCIssuer {
+		want = issuer
+	} else {
+		if tenant != "" {
+			want = fmt.Sprintf(GitHubTenantOIDCIssuer, tenant)
+		} else {
+			want = GitHubOIDCIssuer
+		}
+	}
+
+	certIssuer := attestation.VerificationResult.Signature.Certificate.Extensions.Issuer
+	if !strings.EqualFold(want, certIssuer) {
+		if strings.Index(certIssuer, want) == 0 {
+			return fmt.Errorf("expected Issuer to be %s, got %s -- if you have customized the issuer for your enterprise try using the --cert-oidc-issuer flag with your expected issuer", want, certIssuer)
+		} else {
+			return fmt.Errorf("expected Issuer to be %s, got %s", want, certIssuer)
+		}
+	}
```

Comment:
If the _expected_ value IS NOT an exact match, but IS a substring of the _actual_ value, the user will get the new error message suggesting they use the `--cert-oidc-issuer` flag.